scenarios/mongoose_docker_one_to_one.erl

```diff
@@ -1,5 +1,5 @@
 %%==============================================================================
-%% Copyright 2015 Erlang Solutions Ltd.
+%% Copyright 2019 Erlang Solutions Ltd.
```

Comment:
Should it be changed?

Reply:
How about:
`Copyright (c) 2015-2019 Erlang Solutions Ltd.`
or
`Copyright 2015 Erlang Solutions Ltd.`
`Copyright 2019 Erlang Solutions Ltd.`
?
Like in https://smallbusiness.chron.com/year-copyright-statement-mean-62027.html
section `Several Years`

Reply:
I think List option fits best